cypress/integration/pipeline_page_test.js

```diff
@@ -0,0 +1,52 @@
+describe('Pipeline page tests', () => {
+  context('no access before login', () => {
+    it('redirects to login', () => {
+      cy.visit('/pipelines/pipeline_name');
+      cy.location('pathname').should('eq', '/login');
+    });
+  });
+
+  context('authenticated', () => {
+    beforeEach(() => {
+      cy.fixture('pipeline.sample-computation')
+        .as('pipeline')
+        .then((pipeline) => {
+          cy.server();
+          cy.route('GET', `/flow/v1/*/pipelines/${pipeline.data.name}`, '@pipeline').as(
+            'getPipeline',
+          );
+          cy.route({
```

Comment:
you could check that deleting the pipeline actually triggers the API call to this route

Reply:
Nice suggestion, I added it